entity-framework/core/providers/sql-server/functions.md

```diff
@@ -0,0 +1,173 @@
+dateTime.Date                                               | CONVERT(date, @dateTime)
+dateTime.Day                                                | DATEPART(day, @dateTime)
+dateTime.DayOfYear                                          | DATEPART(dayofyear, @dateTime)
+dateTime.Hour                                               | DATEPART(hour, @dateTime)
+dateTime.Millisecond                                        | DATEPART(millisecond, @dateTime)
+dateTime.Minute                                             | DATEPART(minute, @dateTime)
+dateTime.Month                                              | DATEPART(month, @dateTime)
+dateTime.Second                                             | DATEPART(second, @dateTime)
+dateTime.TimeOfDay                                          | CAST(@dateTime AS time)                              | EF Core 2.2
+dateTime.Year                                               | DATEPART(year, @dateTime)
+DateTimeOffset.Now                                          | SYSDATETIMEOFFSET()
+DateTimeOffset.Today                                        | CONVERT(date, GETDATE())
+DateTimeOffset.UtcNow                                       | SYSUTCDATETIME()
+dateTimeOffset.AddDays(days)                                | DATEADD(day, @days, @dateTimeOffset)
+dateTimeOffset.AddHours(hours)                              | DATEADD(hour, @hours, @dateTimeOffset)
+dateTimeOffset.AddMilliseconds(milliseconds)                | DATEADD(millisecond, @milliseconds, @dateTimeOffset)
+dateTimeOffset.AddMinutes(minutes)                          | DATEADD(minute, @minutes, @dateTimeOffset)
+dateTimeOffset.AddMonths(months)                            | DATEADD(month, @months, @dateTimeOffset)
+dateTimeOffset.AddSeconds(seconds)                          | DATEADD(second, @seconds, @dateTimeOffset)
+dateTimeOffset.AddYears(years)                              | DATEADD(year, @years, @dateTimeOffset)
+dateTimeOffset.Date                                         | CONVERT(date, @dateTimeOffset)
+dateTimeOffset.Day                                          | DATEPART(day, @dateTimeOffset)
+dateTimeOffset.DayOfYear                                    | DATEPART(dayofyear, @dateTimeOffset)
+dateTimeOffset.Hour                                         | DATEPART(hour, @dateTimeOffset)
+dateTimeOffset.Millisecond                                  | DATEPART(millisecond, @dateTimeOffset)
+dateTimeOffset.Minute                                       | DATEPART(minute, @dateTimeOffset)
+dateTimeOffset.Month                                        | DATEPART(month, @dateTimeOffset)
+dateTimeOffset.Second                                       | DATEPART(second, @dateTimeOffset)
+dateTimeOffset.TimeOfDay                                    | CAST(@dateTimeOffset AS time)                        | EF Core 2.2
+dateTimeOffset.Year                                         | DATEPART(year, @dateTimeOffset)
```

Comment:
`year` in DATEPART but `YEAR` in DATEDIFF 🤦‍♂️

Reply:
Different contributor preferences. :trollface: